server.go

```diff
@@ -132,7 +158,7 @@ func (s *StubServer) HandleRequest(w http.ResponseWriter, r *http.Request) {
 	requestData, err := parser.ParseFormString(formString)
 	if err != nil {
 		fmt.Printf("Couldn't parse query/body: %v\n", err)
-		writeResponse(w, r, start, http.StatusInternalServerError, nil)
+		writeResponse(w, r, start, http.StatusInternalServerError, stripeError)
```

Comment:
If the form string is invalid, shouldn't that be a 400 instead of a 500? (But it was like that before, so don't feel obligated to fix in this PR.)